### PR TITLE
Preparation for The Journal of Open Source paper

### PR DESCRIPTION
**Pull request by lucianolorenti**

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      Department of Engineering, University of Padova.
+      Italy
+    index: 1
+date: 20 Dic 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+`CeRULEo` is a Python package designed to train and evaluate regression models for predicting remaining useful life (RUL) of equipment. RUL estimation is a process that uses prediction methods to forecast the future performance of machinery and obtain the time left before machinery loses its operation ability.  The remaining useful life  estimation has been considered as a central
```
Comment (gianantonio) [inline paper/paper.md:26]: add full name for CeRULEo also in the text

File: paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+In order to achieve good performance, RUL regression requires data preparation and feature engineering. Typically, machinery data is provided as time series data from various sensors during operation. The first step in data preparation is often to create a dataset based on run-to-failure cycles. This involves dividing the time series into segments where the equipment starts in a healthy state and ends in a failure state, or is close to failure. The second step of data preparation is preprocessing. While predictive maintenance models can be used in a variety of contexts with different data sources and errors, there are some general techniques that can be applied [@serradilla2022deep].
```
Comment (gianantonio) [inline paper/paper.md:34]: it is not so clear to me what the 'general techniques' comment is about: are we saying that this general techniques is something that we provide with CeRULeo?

File: paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Moreover, `CeRULEo` provides tools for evaluating and comparing predictive maintenance models based on not only traditional regression metrics, but also on their ability to prevent errors and reduce costs. In many cases, the costs of not accurately detecting or anticipating faults can be much higher than the cost of inspections or maintenance due to reduced efficiency, unplanned downtime, and corrective maintenance expenses. In predictive maintenance, it is particularly important to be accurate about the remaining useful life  of equipment near the end of its lifespan, as an overestimation of RUL can have serious consequences when immediate action is required. `CeRULEo` addresses this issue by providing mechanisms for weighting samples according to their importance and asymmetric losses for training models, as well as visualization tools for understanding model performance in relation to true RUL.
+
+
+# Financial Acknowledgement
+
```
Comment (gianantonio) [inline paper/paper.md:46]: 'The Italian Government PNRR iniatiatives 'Partenariato 11: Made in Italy circolare e sostenibile' and 'Ecosistema dell'Innovazione  - iNest' are gratefully acknowledged for partially financing this research activity.'

**Review by gianantonio — CHANGES_REQUESTED** [review]
Small things...

File: paper/paper.md (hunk 2 above)
Comment (lucianolorenti) [reply paper/paper.md]: I added a bit of explanation about the general techniques the text is referring to.

File: paper/paper.md (hunk 1 above)
Comment (lucianolorenti) [reply paper/paper.md]: I added it at the beginning

`CeRULEo`, which stands for Comprehensive utilitiEs for Remaining Useful Life Estimation methOds, is ....

**Review by gianantonio — APPROVED** [review]

**Pull request merged**